README.md

```diff
@@ -65,6 +65,24 @@ python -m unittest test/integration_test_model_bbox.py
 python -m unittest test/integration_test_model_evaluation.py
 ```
 
+##Steps to resolve module related issues on a Windows environment
+Steps to resolve the issue related with 'fiona' and 'geopandas' module
+1)pip uninstall geopandas  //if the package is already installed in the environment
+2)pip install pipwin
+3)pipwin install gdal
+4)pipwin install fiona
+5)pip install geopandas
```

Comment:
I suggest writing this out in code so it can be copied and pasted easier:
```
pip uninstall geopandas  # if the package is already installed in the environment
pip install pipwin
pipwin install gdal fiona
pip install geopandas
```